app/views/api/v2/smart_proxies/main.json.rabl

```diff
@@ -2,11 +2,15 @@ object @smart_proxy
 
 extends "api/v2/smart_proxies/base"
 
-attributes :created_at, :updated_at
+attributes :created_at, :updated_at, :hosts_count
```

Comment:
What is the host count for a smart proxy? How is it defined?

Reply:
It was predefined here: https://github.com/theforeman/foreman/blob/62228620c4265f98bfe5f8eb479a915d87fc94df/app/models/smart_proxy.rb#L51 I am using it to have the same functionality as in the UI.